src/shogun/regression/GLM.cpp

```diff
@@ -0,0 +1,38 @@
+#include <shogun/lib/config.h>
+
+#include <shogun/labels/RegressionLabels.h>
+#include <shogun/mathematics/Math.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/regression/GLM.h>
+#include <utility>
+using namespace shogun;
+
+GLM::GLM() : LinearMachine()
+{
+	init();
+}
+
+void GLM::init()
+{
+	SG_ADD(&m_alpha, "alpha", "alpha parameter", ParameterProperties::HYPER);
```

Comment:
please avoid empty comments/docs/etc as "alpha parameter" for a varialbe called "alpha". If you put a description, why not say what it is or what it does?